tests/portchannel_test.py

```diff
@@ -60,7 +60,30 @@ def test_delete_non_existing_portchannel(self):
         print(result.output)
         assert result.exit_code != 0
         assert "Error: PortChannel0005 is not present." in result.output
-        
+
+    def test_add_portchannel_with_fast_rate(self):
+        runner = CliRunner()
+        db = Db()
+        obj = {'db':db.cfgdb}
+
+        # add a portchannel with fats rate
+        result = runner.invoke(config.config.commands["portchannel"].commands["add"], ["PortChannel0005", "--fast-rate", "True"], obj=obj)
```

Comment:
@globaltrouble please remove all created objects on teardown. Also suggest to have parametrization for `True/False` values

Reply:
@nazariig Parametrization added. Regarding "remove all created objects on teardown" why it is needed? The DB is instantiated for each test from the template  https://github.com/Azure/sonic-utilities/blob/3600639c16079f17089efd655c818022987032bb/tests/mock_tables/config_db.json